Patches/PlayerControlPatch.cs

```diff
@@ -1223,8 +1185,11 @@ public static void Prefix(PlayerControl __instance, [HarmonyArgument(0)] PlayerC
         Logger.Info($"{__instance.GetNameWithRole()} => {target.GetNameWithRole()}{(target.protectedByGuardian ? "(Protected)" : "")}", "MurderPlayer");
 
         if (RandomSpawn.CustomNetworkTransformPatch.NumOfTP.TryGetValue(__instance.PlayerId, out var num) && num > 2) RandomSpawn.CustomNetworkTransformPatch.NumOfTP[__instance.PlayerId] = 3;
-        if (!target.protectedByGuardian && !Doppelganger.DoppelVictim.ContainsKey(target.PlayerId))
+        if (!target.protectedByGuardian && !Doppelganger.DoppelVictim.ContainsKey(target.PlayerId) && !Camouflage.ResetSkinAfterDeathPlayers.Contains(target.PlayerId))
+        {
+            Camouflage.ResetSkinAfterDeathPlayers.Add(target.PlayerId);
```

Comment:
I don’t understand a little, how this can prevent repeated information in the logs?

Reply:
Every murder rpc & killer.murderplayer(target) tiggers this stuff and send rpcsetskin. And rpc set skin produces logs

Reply:
<img width="666" alt="image" src="https://github.com/0xDrMoe/TownofHost-Enhanced/assets/139348239/e31a1787-570a-4d8f-a34b-fe18df799622">